rackcity/models/asset.py

```diff
@@ -55,7 +55,8 @@ def validate_hostname_uniqueness(value, asset_id, change_plan, related_asset):
     assets, assets_cp = get_assets_for_cp(change_plan.id)
     matching_assets = assets_cp.filter(hostname=value, change_plan=change_plan)
     if (
-        len(matching_assets) > 0
+        value
+        and len(matching_assets) > 0
```

Comment:
CRAZY hostname validation bug for assetCPs -> was flagging empty hostnames as non unique

Reply:
OH DEAR